meetings/2020-09-08.md

```diff
@@ -0,0 +1,27 @@
+# ISIS TC Meeting September 8, 2020 @12PM MST
```

Comment:
This is the day after labor day. Will people be on vacation?

Reply:
Sept 8 works for me.